### ISSUE TITLE
Make --version more standardised

### ISSUE THREAD

**Issue by tseemann** [TGAC/KAT#34]
Best to follow the GNU standard to make parsing easier for things like Galaxy etc

OLD:
```
% kat --version
Kmer Analysis Toolkit (KAT) V2.0.8
```

NEW:
```
% kat --version
kat 2.0.8
```

**Comment by maplesond** [comment]
Ok, I'll make this change in V2.1.

**Comment by maplesond** [comment]
Fixed in head of develop branch, will be in V2.1.

**Comment by maplesond** [comment]
Should be fixed in the latest release: https://github.com/TGAC/KAT/releases/tag/Release-2.1.0.  Please reopen if there are any issues.
